app/src/main/java/com/github/se/eventradar/ui/home/Home.kt

```diff
@@ -69,247 +71,247 @@ fun HomeScreen(
     }
   }
 
-  ConstraintLayout(modifier = Modifier.fillMaxSize().testTag("homeScreen")) {
-    val (logo, tabs, searchAndFilter, filterPopUp, eventList, eventMap, bottomNav, viewToggle) =
-        createRefs()
-    Row(
-        modifier =
-            Modifier.fillMaxWidth()
-                .constrainAs(logo) {
-                  top.linkTo(parent.top, margin = 32.dp)
-                  start.linkTo(parent.start, margin = 16.dp)
-                }
-                .testTag("logo"),
-        verticalAlignment = Alignment.CenterVertically) {
-          Image(
-              painter = painterResource(id = R.drawable.event_logo),
-              contentDescription = "Event Radar Logo",
-              modifier = Modifier.size(width = 186.dp, height = 50.dp))
-        }
-    TabRow(
-        selectedTabIndex = getTabIndexFromTabEnum(uiState.tab),
-        modifier =
-            Modifier.fillMaxWidth()
-                .padding(top = 8.dp)
-                .constrainAs(tabs) {
-                  top.linkTo(logo.bottom, margin = 16.dp)
-                  start.linkTo(parent.start)
-                  end.linkTo(parent.end)
+  Scaffold(
+      modifier = Modifier.testTag("homeScreen"),
+      topBar = {
+        Row(
+            modifier = Modifier.fillMaxWidth().padding(top = 32.dp, start = 16.dp).testTag("logo"),
+            verticalAlignment = Alignment.CenterVertically) {
+              Image(
+                  painter = painterResource(id = R.drawable.event_logo),
+                  contentDescription = "Event Radar Logo",
+                  modifier = Modifier.size(width = 186.dp, height = 50.dp))
+            }
+      },
+      bottomBar = {
+        BottomNavigationMenu(
+            onTabSelected = navigationActions::navigateTo,
+            tabList = TOP_LEVEL_DESTINATIONS,
+            selectedItem = getTopLevelDestination(Route.HOME),
```

Comment:
Would be nice to use that notation `selectedItem = getTopLevelDestination(Route.HOME)` in EventDetails & EventSelectTicket too ! (It was my mistake when I've created those files but if you could change it, I would be grateful 🙏 )